docs/meson.build

```diff
@@ -30,7 +41,7 @@ all_doc_target = custom_target(
         '@INPUT@'
     ],
     build_by_default: true,
-    input: vala_files,
+    input: [vala_files, config_file],
```

Comment:
Why are you adding `config_file` here? This looks like an unrelated change.

Reply:
I split the change to https://github.com/elementary/mail/pull/795

Reply:
Awesome, thanks for clarifying and splitting off as a separate change.